File: VB/Dashboard_StringConcatAggregate/Form1.vb
Imports DevExpress.XtraBars.Ribbon

Namespace Dashboard_StringConcatAggregate
	Partial Public Class Form1
		Inherits RibbonForm

		Public Sub New()
			InitializeComponent()
			dashboardDesigner1.CreateRibbon()
			dashboardDesigner1.LoadDashboard("..\..\Data\StringConcatAggregate.xml")
		End Sub
	End Class
End Namespace

File: CS/Dashboard_StringConcatAggregate/Form1.cs
using DevExpress.XtraBars.Ribbon;

namespace Dashboard_StringConcatAggregate {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            dashboardDesigner1.LoadDashboard(@"..\..\Data\StringConcatAggregate.xml");
        }
    }
}
